specs/credentialmanagement/index.src.html

```diff
@@ -515,13 +533,25 @@ <h4 id="interfaces-credential-types-credential"><code>Credential</code></h4>
       <code>password</code>, and {{FederatedCredential}} objects have a
       {{[[type]]}} of <code>federated</code>.
     </dd>
-    <dt><dfn attribute>\[[type]]</dfn></dt>
+    <dt><dfn attribute for="Credential">\[[type]]</dfn></dt>
     <dd>
       All {{Credential}} objects have an internal slot named
       {{[[type]]}}, which unsurprisingly contains a string representing
       the type of the credential. This property is exposed to the web via the
       {{Credential/type}} attribute.
     </dd>
+    <dt><code><dfn method for="Credential">isMatch()</dfn></code></dt>
+    <dd>
+      {{Credential}} objects provide filter matching logic within the
+      <code>isMatch()</code> function. This hook allows different {{Credential}}
+      types to define how they will match the values from the {{CredentialRequestFilters}}
+      passed into the {{CredentialContainer/get()}} function. The base class implementation returns
+      <code>true</code>.
+    </dd>
```

Comment:
Nit: You can just add a newline here rather than a new `<dd>`. That will automagically create paragraphs, markdown-style.